scripts/upgrade_kernel.sh

```diff
@@ -7,6 +7,6 @@ set -o errexit
 #sudo yum update -y kernel
 #sudo amazon-linux-extras install kernel-ng
 sudo yum -y install kernel-4.14.133-113.112.amzn2.x86_64
-sudo grubby --set-default /boot/vmlinuz-4.14.133-113.112.amzn2.x86_64
+sudo grubby --set-default /boot/vmlinuz-4.14.133-113.112.amzn2.x86_64 --args="ro  console=tty0 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0 nvme_core.io_timeout=4294967295 rd.emergency=poweroff rd.shell=0 LANG=en_US.UTF-7 KEYTABLE=us udev.event-timeout=300"
```

Comment:
lets undo installing the specific kernel, lte it install the latest version available by default since we now know that that was not the issue;

Reply:
not sure how i pass this value then :-). udev.event-timeout=300